packages/manager/modules/nutanix/src/dashboard/nodes/list/template.html

```diff
@@ -11,8 +11,13 @@
                 class="oui-link"
                 data-ui-sref="{{:: $ctrl.constructor.getNodeDetailsState($row.name) }}"
                 data-ng-bind="$row.iam.displayName"
+                data-ng-if="!$row.isHiddenNode"
             >
             </a>
+            <span
+                data-ng-if="$row.isHiddenNode"
+                data-ng-bind="$row.iam.displayName"
+            ></span>
```

Comment:
It's duplicated with `iam-protected-data` code.

- May be see for fix `iam-protected-data` if it's the root cause
- Or anythings else without duplicate logic